src/helpers/get-ogc-items-stream.ts

```diff
@@ -0,0 +1,25 @@
+import { PagingStream } from '../paging-stream';
+import { getPagingStream } from './get-paging-stream';
+import * as _ from 'lodash';
+import QueryString from 'qs';
+export const getOgcItemsStream = 
+  async (siteUrl: string, collectionKey: string, requestQuery: QueryString.ParsedQs): Promise<PagingStream> => {
+  const requestParams = buildSearchRequestParams(siteUrl, collectionKey, requestQuery);
+  // OGC API can only return 100 results at maximum so
+  // page limit is set to 1 regardless of the actual
+  // value of limit provided in the request query
+  const maxPageLimit = requestQuery.limit && 1;
+  return getPagingStream(
+    requestParams,
+    maxPageLimit
+  );
+};
+
+const buildSearchRequestParams = (siteUrl: string, collectionKey: string, requestQuery: any) => {
```

Comment:
Instead of `requestQuery: any`, try `requestQuery: Record<string, any>` since you are expecting it to be a hash.

Reply:
Also, this function _does_ build the search params, but it seems to me that its primary function is to construct the full request URL. Maybe we make that clear by changing the name of the function to something like `buildSearchRequestUrl` instead.

Reply:
I agree! Changed types and names to `RequestQuery: Record<string, any>` and `buildSearchRequestUrl`.